.github/resources/.actions/cli-runner/action.yml

```
@@ -0,0 +1,130 @@
+name: Slack CLI Installation and Command Runner 
+description: Download and cache the Slack CLI and run the input command 
+
+inputs:
+  command:
+    description: "Slack CLI command to run"
+    type: string
+    default: ""
+    required: true
+  verbose:
+    description: "Verbose flag"
+    type: boolean
+    default: false
+    required: false
+  cli_version:
+    description: "CLI Version"
+    type: string
+    default: "latest"
+    required: false
+  app_id: 
+    description: "App ID"
+    type: string
+    default: ""
+    required: false
+
+outputs:
+  success:
+    description: "Whether the command ran successfully"
+    value: ${{ steps.run-slack-cli-command.outputs.success }}
+  exit_code:
+    description: "Exit code"
+    value: ${{ steps.run-slack-cli-command.outputs.exit_code }}
+  command_executed:
+    description: "Command ran"
+    value: ${{ steps.run-slack-cli-command.outputs.command_executed }}
+  stdout:
+    description: "Command output"
+    value: ${{ steps.run-slack-cli-command.outputs.stdout }}
```

```suggestion
  output:
    description: "Command output"
    value: ${{ steps.run-slack-cli-command.outputs.output }}
```

📫 suggestion: We might want to capture both `stdout` and `stderr` with this?